apiserver/facades/client/keymanager/service.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package keymanager
+
+import (
+	"context"
+	"net/url"
+
+	coressh "github.com/juju/juju/core/ssh"
+	"github.com/juju/juju/core/user"
+)
+
+// KeyManagerService is the interface for the service this facade requires to
+// perform crud operations on a users public ssh keys within a model.
+type KeyManagerService interface {
+	// AddPublicKeysForUser is responsible for adding one or more new public
+	// keys for a user to the current model.
+	AddPublicKeysForUser(context.Context, user.UUID, ...string) error
+
+	// DeleteKeysForUser is responsible removing one ore more keys from a user
+	// within this model. Keys are identified by either the comment,
+	// public key data or the fingerprint.
+	DeleteKeysForUser(context.Context, user.UUID, ...string) error
+
+	// ImportPublicKeysForUser is responsible for importing keys against a Juju
+	// user for the current model from a third party source identified in the
+	// url.
```

Comment:
Do we really need this one 3 letter word on its own line :-)

Reply:
We do :)

Reply:
Why? The line above even juts out almost enough to accommodate it. Looks butt ugly IMO the way it is :-)